scripts/ci/check_compliance.py

```diff
@@ -213,7 +213,12 @@ def run(self):
 
         except subprocess.CalledProcessError as ex:
             output = ex.output.decode("utf-8")
-            self.add_failure(output)
+            if re.search("[1-9][0-9]* errors,", output):
```

Comment:
This doesn't belong in this PR.  It should be in a separate PR, whether or not there's an issue describing the problem.

Reply:
I've removed this commit and will create a separate PR for that.